main.go

```diff
@@ -65,6 +67,7 @@ func main() {
 		log.Err(err).Msg("can't resolve environment variables")
 		os.Exit(1)
 	}
+	log.Debug().Msgf("Resolved config: %s", str.ToString(config))
```

Comment:
Consider structuring too:
```suggestion
	log.Debug().
		Str("config", str.ToString(config)).
		Msg("Resolved config")
```